packages/angular-test-app/src/preview-examples/modal-form-ix-button-submit.html

```diff
@@ -0,0 +1,37 @@
+<ix-button
+  (click)="openModal()"
+  (keydown.enter)="openModal()"
+  (keydown.space)="openModal()"
+>
+  Show modal
+</ix-button>
+
+<ng-template #customModal let-modal>
+  <ix-modal>
+    <ix-modal-header> Create Resource </ix-modal-header>
+    <ix-modal-content>
+      <form
+        id="create-resource-form"
+        #formRef="ngForm"
+        class="form"
+        (ngSubmit)="onSubmit(formRef)"
+      >
+        <div>
+          <ix-input label="Name:" type="text" name="name" ngModel></ix-input>
+        </div>
```

Comment:
```suggestion
        <ix-input label="Name:" type="text" name="name" ngModel></ix-input>
```

Why do we need this additional element around? Same happens inside other examples too

Reply:
updated it